src/inspector_agent.h

```diff
@@ -75,13 +75,14 @@ class Agent {
   node::Environment* parent_env_;
 
   uv_async_t data_written_;
+  uv_async_t io_thread_req_;
   inspector_socket_t* client_socket_;
   blink::V8Inspector* inspector_;
   v8::Platform* platform_;
-  std::vector<blink::protocol::String16> message_queue_;
+  std::vector<std::string> message_queue_;
+  std::vector<std::string> outgoing_message_queue_;
```

Comment:
Should clean the queue upon disconnect? Or should we serialize connect / disconnect using the same queue?